Specify that funds locked in FB won't be included in offer (#42)
joinmarket-webui/jamdocs: docs/interface/03-earn.md

Fidelity bonds improve the privacy guarantees of the whole system and increase
your chance of being chosen as a [market maker][maker] drastically.

Be aware that funds locked in a Fidelity Bond will **not** be included in your offer amount,

> so remember to leave some funds unlocked order to partecipate as a market maker.

Reviewer: ```suggestion
so remember to keep additional funds in order to participate as a market maker.
```
